content/en/_index.md

```diff
@@ -933,6 +933,12 @@ We prefer to enforce style using standard linters that are popular in
 the community.
 
 {{< content-tabpane >}}
+{{< content-tab header="Java" >}}
+
+Follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html).
+Tooling available at [google/google-java-format](https://github.com/google/google-java-format).
```

Comment:
Do we want to capture how we run lint tests? We use checkstyle with Google style: https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml, which is built into our [repo tooling](https://github.com/GoogleCloudPlatform/java-repo-tools) in order to run: `mvn -P lint checkstyle:check`

Reply:
We don't want to include things that have repo dependencies. Mentioning tooling is a bit at the edge. The specific instructions on commands to run should stay in the repo as part of the specific project process.